code/src/App.js

```diff
@@ -1,9 +1,83 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import ThoughtForm from './components/ThoughtForm'
+import ThoughtList from './components/ThoughtList'
+
+import { API_URL, LIKES_URL } from './reusable/urls'
+
 
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState ([])
+  const [newThought, setNewThought] = useState('')
+
+  useEffect(() => {
+    fetchThoughtList()
+  }, [])
+
+  const fetchThoughtList = () => {
+    fetch (API_URL)
+      .then(response => response.json())
+      .then(thoughts => setThoughtList(thoughts))
+      .catch(error => console.error(error))
+  }
+
+  const handleNewThoughtChange = (event) => {
+    setNewThought(event.target.value);
+  }
+
+  const handleSubmitThought = (event) => {
+    event.preventDefault()
+
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ message: newThought })
+    };
+
+    fetch(API_URL, options)
+      .then(response => response.json())
+      .then(receivedNewThought => setThoughtList([receivedNewThought, ...thoughtList]))
+  //  .then(() => fetchThoughtList)     
```

Comment:
Interesting 🤔 I experienced some problems with this too. Now, it seems to be working on Netlify but not locally, when I try it out in my project.

Reply:
`() => fetchThoughtList()` you forgot the `()`